docs/index.html

```diff
@@ -35,7 +35,11 @@
       href="https://cdnjs.cloudflare.com/ajax/libs/prism/1.20.0/themes/prism-tomorrow.min.css"
       rel="stylesheet"
     />
-
+    <link 
+      href="https://cdn.jsdelivr.net/npm/fork-corner/dist/fork-corner.min.css"
```

Comment:
Is it set as `position: fixed` or `absolute`?

Reply:
It is set as absolute and the parent is relative in position and I position it on top left side of the homepage. Like this one.
![image](https://user-images.githubusercontent.com/15052701/103411972-ff533280-4bad-11eb-9688-24269f3ef0c5.png)

Reply:
Can we make it smaller? I think it's a bit big.

Reply:
Yes! the good thing is that this fork-corner I made is the same as animate.css the way you customize it. Just add ``fc-size-small`` let me do the changes.